File: bin/compassmate-web/web/app/model/AttributeAlias.js
 

Ext.define('Compassmate.model.AttributeAlias', {
    extend: 'Ext.data.Model',
    identifier: 'negative',

    fields: [{
        name: 'id',
        type: 'int'
    }, {
        name: 'deviceId',
        type: 'int'
    }, {
        name: 'attribute',
        type: 'string'
    }, {
        name: 'alias',
        type: 'string'
    }]
});

File: src/mn/compassmate/model/AttributeAlias.java
package mn.compassmate.model;

public class AttributeAlias extends BaseModel {

    private long deviceId;

    public long getDeviceId() {
        return deviceId;
    }

    public void setDeviceId(long deviceId) {
        this.deviceId = deviceId;
    }

    private String attribute;

    public String getAttribute() {
        return attribute;
    }

    public void setAttribute(String attribute) {
        this.attribute = attribute;
    }

    private String alias;

    public String getAlias() {
        return alias;
    }

    public void setAlias(String alias) {
        this.alias = alias;
    }

}
